google-cloud-storage/pom.xml

```diff
@@ -156,6 +156,11 @@
       <artifactId>easymock</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-core</artifactId>
+      <scope>test</scope>
+    </dependency>
```

Comment:
Can you split out mockito tests into a separate PR because it got a bit confusing in review? Apologies for churn on this.

Reply:
Agree. Done #284